### ISSUE TITLE
Quad and Text not affected by Order (DX11.Layer)

### ISSUE THREAD

**Issue by tebjan** [mrvux/dx11-vvvv#313]
only shaders seem to work:

[LayersNotOrdering.zip](https://github.com/mrvux/dx11-vvvv/files/1025758/LayersNotOrdering.zip)

**Comment by mrvux** [comment]
Yes they did not get implemented there (neither validators) indeed
